src/ucm/cuda/replace.c

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "cudamem.h"
+
+#include <ucm/event/event.h>
+#include <ucm/util/log.h>
+#include <ucm/util/reloc.h>
+#include <ucs/sys/compiler.h>
+#include <ucs/sys/preprocessor.h>
+#include <ucs/type/component.h>
+#include <pthread.h>
+
+
+#define MAP_FAILED ((void*)-1)
```

Comment:
is it required? MAP_FAILED is defined in <sys/mman.h>, and MAP_FAILED is not used here